src/ReporterOrchestrator.ts

```diff
@@ -37,7 +38,7 @@ export default class ReporterOrchestrator {
       log.warn(`No reporter configured. Please configure one or more reporters in the (for example: reporter: 'progress')`);
       this.logPossibleReporters();
     }
-    return new BroadcastReporter(reporters);
+    return new BroadcastReporter(reporters) as any;
```

Comment:
This is the magic, an `any` cast basically tells TypeScript that we know best and it in fact *is* a `StrictReporter`.